### DISCUSSION TITLE
Can't move subtitles below video

### DISCUSSION THREAD

**Discussion by billbaldwin2** [smplayer-dev/smplayer#637] [General]
Version: 22.7.0 (revision 10091) (64-bit)

I'm working with a DVD rip where the movie is 2.39 : 1. So there's plenty of black space below the video to put the subtitles. I can move the titles up using the R key, but the T key won't move them any lower than the bottom of the video area.

What I have tried both individually and in all combinations:

Preferences -> General -> Video -> Add black borders for subtitles by default
Preferences -> General -> Video -> Add black borders on fullscreen
Preferences -> Subtitles -> Use ASS library (tried checked and unchecked)

Anything I'm missing?

**Comment by Xenos71** [comment]
Hi, 

try the following:

Preferences --> Subtitles --> Font and colors:
Check "Use custom style", set Vertical margin around 50 and see if anything changes.

I use these settings (and the ones you mentioned about black borders) and I get subs displayed on the black area below the image.

**Reply by billbaldwin2** [reply]
I had such hopes for this, but I couldn't get it to work! I tried 50, 99 and, just to be sure -99--all with and without the ASS library enabled. The subtitles refuse to move below the visible picture. I really appreciate the suggestion, though.

**Reply by Xenos71** [reply]
Strange... I have mpv selected as the multimedia engine (Preferences --> General --> General -> Multimedia engine). 
If you have something else selected, give it a try.
